tests/aws/services/events/test_events_integrations.py

```diff
@@ -237,6 +237,251 @@ def test_put_events_with_target_lambda(create_lambda_function, cleanups, aws_cli
     assert actual_event["detail"] == EVENT_DETAIL
 
 
+@markers.aws.validated
+@pytest.mark.skipif(is_v2_provider(), reason="V2 provider does not support this feature yet")
+def test_put_events_with_target_lambda_list_entry(
+    create_lambda_function, cleanups, aws_client, clean_up, snapshot
+):
+    rule_name = f"rule-{short_uid()}"
+    function_name = f"lambda-func-{short_uid()}"
+    target_id = f"target-{short_uid()}"
+    bus_name = f"bus-{short_uid()}"
+
+    # clean up
+    cleanups.append(lambda: aws_client.lambda_.delete_function(FunctionName=function_name))
+    cleanups.append(lambda: clean_up(bus_name=bus_name, rule_name=rule_name, target_ids=target_id))
```

Comment:
definitely, this is on my plate, to add proper fixtures for events test, since a lot of the tests currently also don't properly clean up after themselves.